recipes/perf/all/conanfile.py

```diff
@@ -0,0 +1,42 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.35.0"
+
+class Perf(ConanFile):
+    name = "perf"
```

Comment:
In most cases I don't like such generic names at all, but it might be okay in this case 🤔

Reply:
Me too. Taking a look on Ubuntu packages, Perf is part of Linux tools and this one is the original Perf, so it looks good using it as "perf".